tests/common/test_cgroupconfigurator.py

```diff
@@ -556,6 +562,127 @@ def mock_popen(command, *args, **kwargs):
 
                     self.assertEqual(expected_output.format("very specific test message"), process_output)
 
+    def test_set_extension_services_cpu_memory_quota(self):
```

Comment:
NIT: Rename test name. We generally follow this naming standard for test names - `test_it_should_*`

Reply:
updated all test cases.